test/WebJobs.Script.Tests/Configuration/HttpWorkerOptionsSetupTests.cs

```diff
@@ -140,6 +140,51 @@ public void InValid_HttpWorkerConfig_Throws_Exception(string hostJsonContent)
             }
         }
 
+        [Theory]
+        [InlineData(@"{
+                    'version': '2.0',
+                    'customHandler': {
+                            'description': {
+                                'defaultExecutablePath': '%TestEnv%',
+                                'defaultWorkerPath': '%TestEnv%'
+                            },
+                        'port': 0
+                        },
+                    }", 0)]
+        [InlineData(@"{
+                    'version': '2.0',
+                    'customHandler': {
+                            'description': {
+                                'defaultExecutablePath': '%TestEnv%',
+                                'defaultWorkerPath': '%TestEnv%'
+                            },
+                        'port': 1234567
+                        },
+                    }", 1234567)]
+        public void CustomHandlerConfig_InvalidPort_ExpandEnvVars(string hostJsonContent, int value)
+        {
+            HttpWorkerOptions options = new();
+
+            try
+            {
+                Environment.SetEnvironmentVariable("TestEnv", "TestVal");
+                File.WriteAllText(_hostJsonFile, hostJsonContent);
+                var configuration = BuildHostJsonConfiguration();
+                HttpWorkerOptionsSetup setup = new(new OptionsWrapper<ScriptJobHostOptions>(_scriptJobHostOptions), configuration, _testLoggerFactory, _metricsLogger, _environment);
+                setup.Configure(options);
+                Assert.Equal("TestVal", options.Description.DefaultExecutablePath);
```

Comment:
Consider using AwesomeAssertions


```suggestion
                options.Description.DefaultExecutablePath.Should().Be("TestVal");
```